draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -2276,7 +2276,16 @@ security properties required by applications.
 strict mode, and when supported by the client. It is, however, prudent to ensure
 that this designation is not the extent of a site's defense against CSRF, as
 same-site navigations and submissions can certainly be executed in conjunction
-with other attack vectors such as cross-site scripting.
+with other attack vectors such as cross-site scripting or abuse of page
+redirections.
+
+Understanding how and when a request is considered same-site is also important
+in order to properly design a site for SameSite cookies. For example, if a
+top-level request is made to a sensitive page that request will be considered
```

Comment:
Nit: There's nuance here, insofar as "top-level request" includes user-initiated navigation from the address bar or right-click menu weirdness.

Reply:
Of course there is :)

Changing it to

`For example, if a user clicks a link and a top-level request is made....`

Should sidestep that problem, wdyt?

Reply:
👍